tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -53,9 +56,41 @@ def setup(self):
         self.webdriver = self.parent.webdriver
 
     def execute_script(self, script, asynchronous=False, args=None):
+        """
+        This method executes JavaScript and if `asynchronous` is True, it waits for the result.
+        """
         method = self.webdriver.execute_async_script if asynchronous else self.webdriver.execute_script
         return method(script, args=args)
 
+    async def async_execute_script(self, script, context, args=None):
```

Comment:
I am thinking if async_execute_script is a good name since the execute_script can also be "async" by setting asynchronous to True. Perhaps we could find a better name but not blocking.

Reply:
Do you have a good idea for the name?

Reply:
Not really, I think it's probably the best name possible and kind of appears to be common in python codebases to prefix `async def` with `async_`